package.json

```diff
@@ -9,13 +9,13 @@
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint ./src --report-unused-disable-directives --max-warnings 5",
-    "prepackage-debian": "npm run build",
-    "package-debian": "node scripts/package-linux.mjs --debian ./debian/",
-    "prepackage-linux": "npm run build",
-    "package-linux": "node scripts/package-linux.mjs ./build/",
-    "prepackage-conan": "npm run package-linux",
+    "prepackage-debian": "npm run build && cd server && cross build --target armv7-unknown-linux-gnueabihf --release",
+    "package-debian": "node scripts/package-linux.mjs --debian ./debian/ armv7-unknown-linux-gnueabihf",
+    "prepackage-linux": "npm run build && cd server && cross build --target aarch64-unknown-linux-gnu --release",
+    "package-linux": "node scripts/package-linux.mjs ./build/ aarch64-unknown-linux-gnu",
     "package-conan": "node scripts/package-conan.mjs",
-    "preversion": "eslint ./src --report-unused-disable-directives --max-warnings 5"
+    "preversion": "eslint ./src --report-unused-disable-directives --max-warnings 5",
+    "postversion": "node scripts/apply-version.mjs"
```

Comment:
will have to check if that would be picked up by the pending changeset PR or if we just need to integrate that into the `release` script that gets called.

Reply:
Yeah, I had that same thought when I looked at the changeset PR